lib/SILOptimizer/IPO/CrossModuleOptimization.cpp

```diff
@@ -138,10 +139,10 @@ class InstructionVisitor : public SILCloner<InstructionVisitor> {
 
 private:
   CrossModuleOptimization &CMS;
-
```

Comment:
Please update the comment for InstructionVisitor. Something like "Visitor for checking if all types of an instruction are serializable and making used types of an instruction inlinable."